backend/api/app/application/services/survey_service.py

```diff
@@ -41,4 +47,27 @@ def create(self, survey_id, email):
 
         SurveyResponsesRepository.add(survey_response)
 
-        return survey_response
\ No newline at end of file
+        return survey_response
+    
+    def count_responses_by_survey_id(self, survey_id: str) -> dict:
+        yes: int = SurveyResponsesRepository.get_yes_count(survey_id)
+        no: int = SurveyResponsesRepository.get_no_count(survey_id)
+        maybe: int = SurveyResponsesRepository.get_maybe_count(survey_id)
+        
+        return {
+            "yes": yes or 0,
+            "no": no or 0,
+            "no response": maybe or 0
```

Comment:
As you will see in comment down below, we tend to use camelCase (or dashes) for dto fields.